redirect/routes.php

```diff
@@ -210,8 +210,7 @@ class Route {
         ],
         'gad'              => [
             'description' => 'Grundlegende Algorithmen und Datenstrukturen',
-            'target'      => 'https://www.moodle.tum.de/course/view.php?id=53358',
-            'moodle_id'   => '53358',
+            'target'      => 'https://artemis.ase.in.tum.de/courses/119',
```

Comment:
I guess they don't use moodle anymore?

Reply:
GAD still has a moodle course, but its only a landing page, all the lectures and excercises are on Artemis